locals.tf

```diff
@@ -3,15 +3,27 @@ locals {
     enabled  = var.highly_available == true ? true : false
     location = "eu"
   }
+  default_insights_config = {
+    query_string_length     = 1024
+    record_application_tags = false
+    record_client_address   = false
+  }
```

Comment:
... defining some locals ...

Reply:
Which is why I set a default insights config here (merely a helper variable).